sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3249,6 +3249,11 @@ object GpuOverrides extends Logging {
       (c, conf, p, r) => new TypedImperativeAggExprMeta[ApproximatePercentile](c, conf, p, r) {
 
         override def tagAggForGpu(): Unit = {
+          if (!conf.isIncompatEnabled) {
```

Comment:
Rather than doing this check explicitly, we should be using the `incompat` method on the meta to do this check and also generate documentation.